requirements/edx/base.in

```diff
@@ -75,7 +75,7 @@ edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions==2.3.1
 edx-django-utils
 edx-drf-extensions
-edx-enterprise
+edx-enterprise==1.6.9
```

Comment:
Why add this pin?

Reply:
The upgrade breaks several tests due to a model change, I figure that should be sorted out in a separate PR.